src/goto-instrument/accelerate/disjunctive_polynomial_acceleration.h

```diff
@@ -38,14 +38,16 @@ class disjunctive_polynomial_accelerationt:public loop_accelerationt
     goto_functionst &_goto_functions,
     goto_programt &_goto_program,
     natural_loops_mutablet::natural_loopt &_loop,
-    goto_programt::targett _loop_header):
+    goto_programt::targett _loop_header,
+    const satcheck_infot &satcheck_info):
     symbol_table(_symbol_table),
     ns(symbol_table),
     goto_functions(_goto_functions),
     goto_program(_goto_program),
     loop(_loop),
     loop_header(_loop_header),
-    utils(symbol_table, goto_functions, loop_counter)
+    utils(symbol_table, goto_functions, loop_counter),
+    satcheck_info_(satcheck_info)
```

Comment:
Underscores at the beginning, underscores at the end... -- this is super-inconsistent. Please follow the style of the code that you are editing instead of making it incoherent on purpose!